CHANGELOG.md

```diff
@@ -0,0 +1,450 @@
+All notable changes to this project will be documented in this file.
+We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
+
+## Unreleased
+
+### Added
+- **capital-framework:** Add atomic-component dependency to main package.json.
+
+### Changed
+- **cf-expandables:** Update atomic-component dependency to 1.3.2.
+- **cf-tables:** Update atomic-component dependency to 1.3.2.
+
+### Removed
+- **all components:** All individual component lockfiles.
+
+
+## 4.9.0 - 2017-09-06
+
+### Added
+- **cf-notification:** 1.0 added.
+- **cf-modals:** 1.0 added.
```

Comment:
Huh? The cf-modals PR hasn't been merged.

Reply:
Hmm I bet when @anselmbradford [recovered the changelog](https://github.com/cfpb/capital-framework/pull/589/commits/d2897022b5c46da6b01a66416aeeb465a63e86fa) he copied it from his pending modals PR. I'll remove that line.